pkg/headers/header_contents.go

```diff
@@ -11,3 +11,7 @@ const APIVersionKey = "X-GitHub-Api-Version"
 
 // TODO(kfcampbell): get the version from the generated code somehow
 const APIVersionValue = "2022-11-28"
+
+const XRateLimitRemainingKey = "X-Ratelimit-Remaining"
+const XRateLimitResetKey = "X-Ratelimit-Reset"
+const RetryAfterKey = "Retry-After"
```

Comment:
Do we want to go ahead and add all of the potential rate limit headers here:
`x-ratelimit-limit`
`x-ratelimit-used`
`x-ratelimit-resource`

And perhaps expose the values as properties or logged items - to help with debugging, usage?